macros/dmt_get_test_sql.sql

```diff
@@ -28,11 +28,26 @@
             {% set ns.test_sql = render(ns.test_sql)|replace(ns.rendered_keys[k], v) %}
         {% endfor %}
 
-        {% set mock_model_relation = dbt_datamocktool._get_model_to_mock(
+        {# Store model result for visibility in case of unit test failure #}
+        {% set identifier_name = model.name %}
+        {% if test_case %}
+            {% set identifier_name = identifier_name ~ '__test_case_'~test_case %}        
+        {% endif %}
+        {# Note: possible to hardcode desired database.schema to store the mocked model output #}
+        {% set mock_model_relation = api.Relation.create(
+                    database=model.database,
+                    schema=model.schema,
+                    identifier=identifier_name,
+                    type='view') %}

```

Comment:
This `model.database/schema` will retrieve the value from the model where the test is being applied.
Alternatively, I wanted to use the default value as defined in the `dbt_project.yaml`, for the seeds. Any idea how to do this?

In my use case, I was able to hard-code the value, but would be nice to have a more elegant solution